src/js/stores/UserStore.js

```diff
@@ -1,73 +1,77 @@
-import {Store} from 'mesosphere-shared-reactjs';
+import EventEmitter from 'events';
 
 import AppDispatcher from '../events/AppDispatcher';
-
 import {
   REQUEST_USER_CREATE_SUCCESS,
   REQUEST_USER_CREATE_ERROR,
   REQUEST_USER_DELETE_SUCCESS,
   REQUEST_USER_DELETE_ERROR,
   SERVER_ACTION
 } from '../constants/ActionTypes';
-
 import {
   USER_CREATE_SUCCESS,
   USER_CREATE_ERROR,
   USER_DELETE_SUCCESS,
   USER_DELETE_ERROR
 } from '../constants/EventTypes';
-
 import UsersActions from '../events/UsersActions';
 
 /**
  * This store will keep track of users and their details
  */
-const UserStore = Store.createStore({
-  storeID: 'user',
+class UserStore extends EventEmitter {
```

Comment:
There is no need to, since it is never used to `.get(...)` or `.set(...)` anything, which is the only thing that the `BaseStore` as of now exposes (same functionality as the GetSetMixin).